runtime/src/iree/tooling/trace_replay.c

```diff
@@ -532,8 +537,13 @@ static void iree_trace_replay_write_element(
     IREE_TRACE_REPLAY_WRITE_ELEMENT_CASE(UINT_16, uint16_t)
     IREE_TRACE_REPLAY_WRITE_ELEMENT_CASE(UINT_32, uint32_t)
     IREE_TRACE_REPLAY_WRITE_ELEMENT_CASE(UINT_64, uint64_t)
+    // clang-format off
+    case IREE_HAL_ELEMENT_TYPE_FLOAT_16:
+      *(uint16_t*)dst = iree_math_f32_to_f16((float)value);
+      break;
     IREE_TRACE_REPLAY_WRITE_ELEMENT_CASE(FLOAT_32, float)
     IREE_TRACE_REPLAY_WRITE_ELEMENT_CASE(FLOAT_64, double)
+    // clang-format on
```

Comment:
shouldn't the `clang-format on` be moved 2 lines up, right after the `break;` ?

Reply:
if the clang-format is turned on after the `break;`, it indents the line at 544 and 545 by two spaces to the right and aligns it with the `break;`.